Practice2/src/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8"/>
+    <title>My First Parcel App</title>
+    <link rel="stylesheet" href="main.css">
+    <title>Hofmann Kit</title>
```

Comment:
Missed setting viewport for responsive. Please refer link here: https://www.w3schools.com/html/html_responsive.asp